### DISCUSSION TITLE
forwardPorts in devcontainer.json not taking effect

### DISCUSSION THREAD

**Discussion by vadi2** [community/community#26845] [Codespaces]
<p>I’ve got the following forwardports in my <a href="https://github.com/Mudlet/Mudlet/blob/b5502eeb3bd08a3bc332609d7fba6204e1550ac6/.devcontainer/devcontainer.json#L24" rel="noopener nofollow ugc">devcontainer.json</a>:</p>
<pre><code class="lang-auto">"forwardPorts": [5900, 5901, 6080],
</code></pre>
<p>Yet when I launch the codespace, the codespaces extension doesn’t think any are forwarded:</p>
<p><img src="https://user-images.githubusercontent.com/110988/181100262-980319cf-2e8b-4f61-8621-61f5c337aa00.png" alt="image" data-base62-sha1="sXbCrz0YgER2wXyQ1UE2u6m3CEV" width="351" height="206"></p>

**Comment by lostintangent** [comment, marked as answer]
<p>Thanks for reporting this! We had a regression in the UI, that prevents these configured ports from sometimes showing up. We’ll have a fix out for that early this week :+1: In the meantime, if you spin up a server from the terminal, and cmd/ctrl+click it’s URL, that should still work. Additionally, if you need to retrieve a URL for a specific port, you can run “npx codespaces-port ” to grab it.</p>
<p>Otherwise, we’ll update this topic as soon as the UI is fixed next week. Stay tuned!</p>
